docs/pages/reference/signature-algorithms.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Signature Algorithms
+h1: Signature Algorithms Reference
+description: "Signature algorithms used in Teleport."
+---
+
+An important role of the Teleport Auth Service is issuing short-lived SSH and
+TLS certificates to users and hosts that allow all connections to be
+authenticated, authorized, and encrypted.
+This page describes the various cryptographic signature algorithms used to sign
+each kind of certificate issued by Teleport.
+
+Historically, every Teleport-issued certificate was signed by a 2048-bit RSA key.
+RSA is one of the oldest and most well-known asymmetric cryptography algorithms.
+While it is still considered secure (as long as your keys are large enough),
+modern alternatives based on Elliptic-Curve Cryptography offer much better
+performance with smaller keys and better security.
+
+## Signature algorithm suites
+
+Unilaterally changing the signature algorithms used in Teleport would be a
+breaking change for our users, as certain environments may expect or rely on RSA
+signatures.
+Some users may also require different algorithms to be used if they are looking
+for FIPS compliance or they are storing key material in a Hardware Security
+Module (HSM) or Key Management Service (KMS) that only supports certain
+algorithms.
+
+To allow new and existing Teleport users to opt-in to the new signature
+algorithms when they are ready, we introduced the concept of "signature
+algorithm suites".
+By selecting a single algorithm suite, Teleport admins can control all of the
+cryptographic signature algorithms used across their cluster.
+
+### `legacy`
+
+The `legacy` suite identifies the original Teleport behavior where all
+signatures are based on 2048-bit RSA keys.
+Teleport clusters created on versions prior to 17.0.0 effectively have always
+used the `legacy` suite and this will not automatically change when they upgrade
+to newer versions.
+
+We recommend that users upgrade to one of the newer suites when they are able.
+
+### `balanced-v1`
+
+The `balanced-v1` suite is the default suite for self-hosted clusters created
+after version 17.0.0.
+It is our recommended suite for most self-hosted users.
+When this suite is configured, Ed25519 is used for all SSH certificates.
+ECDSA with the NIST P-256 curve is used for all TLS certificates.
+
+RSA is still used where compatibility with common third-party software that
+Teleport integrates with is known to be unable to support non-RSA algorithms.
+This includes Database Access certificates and certain JSON Web Tokens (JWTs)
```

Comment:
Since we're moving away from describing separate "[Resource] Access" products in favor of "enrolling resources", and already describe the different CAs [in the docs](https://goteleport.com/docs/admin-guides/management/operations/ca-rotation/), I think it would make sense to say "certificates issued by the `db` or `db_client` CAs" (or one of these CAs, depending on what's accurate) instead of "Database Access certificates".